proxy/core/src/main/resources/reference.conf

```diff
@@ -147,7 +147,7 @@ cloudstate.proxy {
         enabled = false
 
         # Passivation timeout for Value Entities
-        passivation-timeout = 30s
+        passivation-timeout = 30s // Keep short for testing purposes
```

Comment:
Are 30s then also a good option for production?
Perhaps we might document that, that the default is not the value to be chosen for production environments.

Reply:
It is true and it will be documented. I am not sure if 30s is a good option for production. Perhaps @pvlugter can give more insight here.